src/test/FridgeTest.java

```diff
@@ -0,0 +1,69 @@
+package duke.test;
+
+import duke.exception.DukeException;
+import duke.fridge.Fridge;
+import duke.ingredient.Ingredient;
+import duke.parser.Convert;
+import org.junit.jupiter.api.Test;
+
+import java.util.Date;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class FridgeTest {
+
+    Fridge fridge = new Fridge();
+    @Test
+    void testPutIngredient() throws DukeException {
+        fridge = new Fridge();
```

Comment:
Names for tests should follow the format **featureUnderTest_testScenario_expectedBehavior()**. Update accordingly for all the testcases